autoload/neomake/makers/ft/bib.vim

```diff
@@ -0,0 +1,15 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#bib#EnabledMakers() abort
+    return []
+endfunction
+
+function! neomake#makers#ft#bib#bibtex() abort
+    return {
+                \ 'exe': 'bibtex',
```

Comment:
`exe` defaults to the name, so could be left out.  But it also does not hurt to keep it.  Just for information.